dev/t3c/run.sh

```diff
@@ -20,6 +20,9 @@
 set -o errexit
 trap '[ $? -eq 0 ] && exit 0 || echo "Error on line ${LINENO} of ${0}"; exit 1' EXIT
 
+cd $TC
+go mod vendor
```

Comment:
`go mod vendor` removes all the contents of the `vendor` directory and re-adds them, so we shouldn't run `go mod vendor` unless it's actually needed.

Everything in the `vendor` directory is tracked in git except for `vendor/golang.org/x`, so maybe only run `go mod vendor` if that directory doesn't exist?

Reply:
I added the check and action to the atc.dev.sh script instead, as that should exist before the components are started without the components needed to check for it or wait on it.